src/api/helpers/map-CIS-countries/map-CIS-country/short-term-cover-is-yes-refer-or-unlisted/index.ts

```diff
@@ -7,12 +7,12 @@ const {
 } = EXTERNAL_API_DEFINITIONS;
 
 /**
- * hasValidShortTermCover
+ * shortTermCoverIsYesReferOrUnlisted
  * Check if a country's "short term cover" flag can apply for insurance online.
  * @param {String} shortTermCover: Short term cover
  * @returns {Boolean}
  */
-const hasValidShortTermCover = (shortTermCover: string): boolean => {
+const shortTermCoverIsYesReferOrUnlisted = (shortTermCover: string): boolean => {
```

Comment:
this name is hard to grasp when looking at it from another file calling it `hasYesReferOrUnlistedShortTermCover`?

Reply:
I think it is clearer to have the property name at the beginning, like other helpers e.g

```
esraClassificationIsStandardHighOrVeryHigh
countryRatingIsAorB
```

So the pattern is:

```
propertyIsX
propertyIsXorY
```

Rather than 

```
hasXorYProperty
hasXYorZProperty
```